source/includes/connection-snippets/x509-snippets/ruby-connection.rb

```diff
@@ -0,0 +1,14 @@
+# begin x509 connection
+require 'mongo'
+
+begin
+  client = Mongo::Client.new(
+    ['<cluster-url>'],
+    auth_mech. mongodb_x509,
+    ssl: true,
```

Comment:
@p-mongo shouldn't this be updated to use tls* option names as well?

Reply:
Yes the tls names would be preferred.